doc/changelog.md

```diff
@@ -5,9 +5,13 @@
 * `qjit` adheres to user-specified `mcm_method` given to the `QNode`.
   [(#798)](https://github.com/PennyLaneAI/catalyst/pull/798)
 
+* `MidCiruitMeasure` now inherits `MidMeasureMP` instead of `HybridOp`, allowing much better integration with PennyLane for
+  workflows with mid-circuit measurements.
+  [(#832)](https://github.com/PennyLaneAI/catalyst/pull/832)
```

Comment:
I didn't even realize Catalyst had an "Internal changes" section in the changelog. Pennylane doesn't have one 🤔

Reply:
Yeah we've added it because there are things that are "user" targeted, and things that are "developer" targeted. The internal changes would mainly affect people poking around our package but not users.